resource-validator/src/main/scala/com/broadinstitute/dsp/resourceValidator/DbReader.scala

```diff
@@ -130,6 +132,14 @@ object DbReader {
          """
       .query[KubernetesClusterToRemove]
 
+  val dataprocClusterWithWorkersQuery =
+    sql"""SELECT DISTINCT c1.id, googleProject, clusterName, rt.cloudService, c1.status, rt.numberOfWorkers, rt.numberOfPreemptibleWorkers
+          FROM CLUSTER AS c1
+          INNER JOIN RUNTIME_CONFIG AS rt ON c1.`runtimeConfigId`=rt.id
+          WHERE rt.cloudService="DATAPROC" AND NOT rt.numberOfWorkers=0 AND rt.numberOfWorkers IS NOT NULL AND NOT c1.status="DELETED"
```

Comment:
Do we catch anomalies where Leo DB indicates zero workers but there are non-zero workers in Google?